ideascube/blog/migrations/0010_escape_summary.py

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+from django.db import migrations
+from django.utils.html import escape
+
+
+def escape_summary(app, schema_editor):
+    Content = app.get_model('blog', 'Content')
+    db_alias = schema_editor.connection.alias
+    for content in Content.objects.using(db_alias).all():
+        content.summary = "<p>{}</p>".format(escape(content.summary))
```

Comment:
So you're wrapping the summary in `<p>...</p>`.

But then in the `content_card.html` template, it's displayed inside of another `<p>`, along with an optional image.

This doesn't seem right.